makejdk-any-platform.sh

```diff
@@ -59,13 +59,14 @@ for i in "$@"; do
   esac
 done
 
-if [ "$OPENJDK_CORE_VERSION" == "jdk9" ]; then
+echo $OPENJDK_CORE_VERSION
```

Comment:
Did you mean to leave this here? It either needs to be removed or double quoted

Reply:
good spot - I'll remove this